lib/proxy_api/tftp.rb

```diff
@@ -38,12 +40,14 @@ def fetch_boot_file(args)
       raise ProxyException.new(url, e, N_("Unable to fetch TFTP boot file"))
     end
 
-    # returns the TFTP boot server for this proxy
+    # returns the TFTP boot server IP for this proxy
     def bootServer
+      # read TFTP servername from proxy setting
       if (response = parse(get("serverName"))) && response["serverName"].present?
-        return response["serverName"]
+        response["serverName"]
```

Comment:
This can actually also be a FQDN. Previously this would be resolved but the current implementation would break.

Reply:
Oh I see, I thought that installer only accepts IPv4 for some reason. Will fix this.

Reply:
If only. I actually wouldn't mind deprecating that and adding a warning in the proxy when it sees a non-IP. Then on the Foreman side we would change the behavior to trust the response if it's an IP rather than doing the ip -> name -> ip dance.